docs/Configuration.md

```diff
@@ -63,3 +63,9 @@ spark.driver.extraClassPath    ${GLUTEN_HOME}/package/target/gluten-<>-jar-with-
 spark.executor.extraClassPath    ${GLUTEN_HOME}/package/target/gluten-<>-jar-with-dependencies.jar
 ######
 ```
+
+Additionally, you can control the configurations of gluten at thread level by local property.
```

Comment:
Assume users have to get a spark context to set local property. Could you add more details for how to set such property? Thanks!